lib/src/descriptor.pb.dart

```diff
@@ -151,11 +153,12 @@ class _ReadonlyFileDescriptorProto extends FileDescriptorProto
     with ReadonlyMessageMixin {}
 
 class DescriptorProto_ExtensionRange extends GeneratedMessage {
-  static final BuilderInfo _i =
-      new BuilderInfo('DescriptorProto_ExtensionRange')
-        ..a<int>(1, 'start', PbFieldType.O3)
-        ..a<int>(2, 'end', PbFieldType.O3)
-        ..hasRequiredFields = false;
+  static final BuilderInfo _i = new BuilderInfo(
+      'DescriptorProto_ExtensionRange',
+      packageName: 'google.protobuf.DescriptorProto')
```

Comment:
This comment is regarding an outdated version of this pull request.
Now we only generate code for one field, but use the fully qualified name instead of the className as the string.

It will of course make the string constants longer adding a package-prefix, but not add another field to the method.